media/js/firefox/sync.js

```diff
@@ -90,15 +84,6 @@
         state = 'Not Firefox';
     }
 
-    // Send page state to GA if it hasn't already been sent in the
-    // getConfiguration callback
-    if (syncCapable === false) {
```

Comment:
It looks like this was the only use of the `syncCapable` variable, so let's remove it elsewhere in the script.